test/unit/touch-enroll-specs.js

```diff
@@ -0,0 +1,169 @@
+// transpile:mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { exec as execNode } from 'child_process';
+import B from 'bluebird';
+import { getUserDefault, setUserDefault, setTouchEnrollKey, setTouchEnrollKeys, getTouchEnrollKeys, touchEnrollMenuKeys, 
+  restoreTouchEnrollShortcuts, NS_USER_KEY_EQUIVALENTS, TOUCH_ENROLL_KEY_CODE } from '../../lib/touch-enroll';
+
+const exec = B.promisify(execNode);
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('touch-enroll.js', () => {
+
+  describe('getUserDefaults(), setUserDefaults()', () => {
+    let domain = 'fakeDomain';
+
+  // Nuke the contents of the domain so it's fresh every test case
+    let clearFakeDomain = async function () {
```

Comment:
What if I broke up the tests into a `touch-enroll-specs.js` which uses mocks/stubs (the way unit tests are supposed to) and then I moved the system altering tests to `touch-enroll-e2e.js`? I just want to have tests in place that verify the correctness of the calls to `exec('default...')`